src/standalone/Selector/SingleSelect.tsx

```diff
@@ -1,14 +1,8 @@
 import React from "react";
-import Selector, { SelectorData, SelectorPropsSingleSelect } from "./Selector";
+import BaseSelector, { BaseSelectorProps } from "./BaseSelector";
 
-const SingleSelect = <Data extends SelectorData>(
-	props: Omit<SelectorPropsSingleSelect<Data>, "multiSelect">
-) => (
-	<Selector<Data, false>
-		{...props}
-		onSelect={props.onSelect}
-		multiSelect={false}
-	/>
-);
+const SingleSelect = (props: BaseSelectorProps) => {
```

Comment:
If SingleSelect doesn't add extra logic remove it and rename the BaseSelector to SingleSelect

Reply:
@TQjona , Yes, SingleSelect doesn't add extra logic. But renaming to BaseSelector to SingleSelect is create bit confusion as MultiSelect also uses the same BaseSelector. So, I think it's better we separate both SingleSelect and MultiSelect.

Reply:
I guess we can keep it like this then for now in case we want to add more logic later on